tests/packages/Microsoft.Bot.Components.Telephony.Tests/Integration/CallTransferTests/CallTransfer_BaseScenario.test.dialog

```diff
@@ -5,12 +5,16 @@
     "triggers": [
         {
             "$kind": "Microsoft.OnBeginDialog",
-            "actions": [
-                {
-                    "$kind": "Microsoft.Telephony.CallTransfer",
-                    "phoneNumber": "+1555443443"
-                }
-            ]
+          "actions": [
+            {
+              "$kind": "Microsoft.Telephony.CallTransfer",
+              "phoneNumber": "+1555443443"
+            },
+            {
+              "$kind": "Microsoft.SendActivity",
```

Comment:
In a real TC call, will we receive this activity? Or will this activity be dropped and the call terminate with an EndOfConversation activity?

Reply:
Good question, I think in real call once we finish call transfer we cant get back to bot control (you might know better :)). I can remove this sendactivity

Reply:
I think the posterior send activity is good, because in telephony channel we should never see this one (since it ended up getting transferred) and in non-telephony channels we should see this (because it should be a no-op)

Reply:
we do get control of the bot in this test framework, even in telephony channel, since we dont do the actual transfer from what I can see